client/src/components/Stars.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const stars = (numOfStars) => {
```

Comment:
It looks as if this is a helper function and not a component itself?  I can't recall, but would the file name then be a lowercase "stars"?

Reply:
You're also right about this. I wasn't really thinking. I would like to change it in the future to an actual component, and handle change. Should I just leave it for now?